<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное окно</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>528</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_main">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_iterator_dir">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {background-color: rgb(122, 88, 200)}
QPushButton{border: 2px solid black}</string>
           </property>
           <property name="text">
            <string>Изучить директорию</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_path">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Выбранная директория:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dir"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_sise_dir">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_count_files">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_count_dirs">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_time">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_sort">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Сортировать по типам файлов:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_filters">
           <property name="styleSheet">
            <string notr="true">QListWidget {border: 1px solid black}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_apply_filters">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {border: 2px solid black}
QPushButton {background-color: rgb(0, 144, 44)}</string>
           </property>
           <property name="text">
            <string>Применить фильтры</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_reset_filters">
           <property name="styleSheet">
            <string notr="true">QPushButton {border: 2px solid black}
QPushButton {background-color: rgb(244, 0, 33)}</string>
           </property>
           <property name="text">
            <string>Сбросить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_count_sorted_files">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <column>
            <property name="text">
             <string>Расширение файла</string>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignJustify|AlignVCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Количество</string>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="horizontalFrame_graphics">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
